pkgs/applications/networking/pyload/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, fetchFromGitHub, fetchpatch, python, gocr, unrar, rhino, spidermonkey }:
+let
+  pythonEnv = (python.withPackages (ps: with ps; [
+    pycurl jinja2 beaker thrift simplejson
+    #optional packages
+    #TODO add Send2Trash
+    #TODO package the other optional ones, see homepage
+    pycrypto feedparser pyqt4
+  ]));
+in
+stdenv.mkDerivation rec {
+  version = "stable";
+  name = "pyLoad-" + version;
+
+  src = fetchFromGitHub {
+    owner = "pyload";
+    repo = "pyload";
+    rev = "stable";
+    sha256 = "13as5z38v4iaq9fziiq5cim3bmllwcf4dhpigh7mvrih93pn90hn";
+  };
+
+  installPhase =
```

Comment:
looking a bit further at the package, it seems it is a Python package that uses `paver`. We have `paver` in `python-packages.nix`. And it's possible to create a `setup.py` with `paver generate_setup`. Doing that should result in a `setup.py` allowing the package to be installed with `buildPythonApplication`.